Traceback (most recent call last):
  File "client.py", line 154, in <module>
  File "socket.pyc", line 228, in meth
socket.error: [Errno 10061] 
